timesketch.conf

```diff
@@ -0,0 +1,29 @@
+# Timesketch configuration
+#
+# The default location for this configuration file in /etc/timesketch.conf
+# If you put it somewhere else you can pass the path to tsctl.py
+# Example:
+#
+# $ tsctl.py -c /path/to/this/timesketch.conf runserver
+
+# Turn off debugging.
+# It is a security risk to have this enabled in production.
+DEBUG = False
+
+# This should be a unique random string. Don't share this with anyone.
```

Comment:
What does this actually secure? Is this the seed for XSRF protection, or something else?